packages/integration-sdk-testing/src/context.ts

```diff
@@ -20,45 +22,60 @@ import {
 import { createMockIntegrationLogger } from './logger';
 
 export type CreateMockExecutionContextOptions<
-  TConfig extends IntegrationInstanceConfig = IntegrationInstanceConfig,
+  TInstanceConfig extends IntegrationInstanceConfig = IntegrationInstanceConfig,
+  TExecutionConfig extends IntegrationExecutionConfig = IntegrationExecutionConfig,
 > =
-  | CreateMockExecutionContextOptionsWithInstanceConfig<TConfig>
-  | CreateMockExecutionContextOptionsWithInstanceConfigFields<TConfig>;
+  | CreateMockExecutionContextOptionsWithConfigs<
+      TInstanceConfig,
+      TExecutionConfig
+    >
+  | CreateMockExecutionContextOptionsWithInstanceConfigFields<
+      TInstanceConfig,
+      TExecutionConfig
+    >;
 
-interface CreateMockExecutionContextOptionsWithInstanceConfig<
-  TConfig extends IntegrationInstanceConfig = IntegrationInstanceConfig,
+interface CreateMockExecutionContextOptionsWithConfigs<
+  TInstanceConfig extends IntegrationInstanceConfig = IntegrationInstanceConfig,
+  TExecutionConfig extends IntegrationExecutionConfig = IntegrationExecutionConfig,
 > {
-  instanceConfig: TConfig;
+  instanceConfig: TInstanceConfig;
+  executionConfig: TExecutionConfig;
```

Comment:
This looks like a breaking change. Should this be optional? Many integrations are using the `createMockExecutionContext` test function and will complain about the `executionConfig` property being omitted. https://github.com/search?q=org%3AJupiterOne+%22createMockExecutionContext%22&type=code